### ISSUE TITLE
Image ghcr.io/home-assistant/raspberrypi4-homeassistant error 404

### ISSUE THREAD

**Issue by ronycohen** [home-assistant/supervised-installer#275]
### OS Version

 Debian GNU/Linux 11 (bullseye)

### System Information

Linux pi4 5.15.84-v8+ #1613 SMP PREEMPT Thu Jan 5 12:03:08 GMT 2023 aarch64 GNU/Linux

### What happened?

I encountered an issue when trying to install Home Assistant to the Debian package.
Steps to recreate:

Step 4: Install the Home Assistant Supervised Debian Package:
wget https://github.com/home-assistant/supervised-installer/releases/latest/download/homeassistant-supervised.deb
dpkg -i homeassistant-supervised.deb

Logs on the supervisor container render an error.

### Machine Type

raspberrypi4-64

### Installer output

```text
23-02-07 19:33:04 ERROR (SyncWorker_1) [supervisor.docker.interface] Can't install ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2: 404 Client Error for http+docker://localhost/v1.42/images/ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2/json: Not Found ("No such image: ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2")
23-02-07 19:33:04 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec
23-02-07 19:33:34 INFO (SyncWorker_2) [supervisor.docker.interface] Updating image ghcr.io/home-assistant/raspberrypi4-homeassistant:landingpage to ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2
23-02-07 19:33:34 INFO (SyncWorker_2) [supervisor.docker.interface] Downloading docker image ghcr.io/home-assistant/raspberrypi4-homeassistant with tag 2023.2.2.
23-02-07 19:34:05 ERROR (SyncWorker_2) [supervisor.docker.interface] Can't install ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2: 404 Client Error for http+docker://localhost/v1.42/images/ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2/json: Not Found ("No such image: ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.2")
```

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by ikifar2012** [comment]
This is usually a DNS issue please check that you can resolve `ghcr.io`

**Comment by ronycohen** [comment]
Hi, I don't have any DNS restrictions...
If it's working on your side then I'll retry.

**Comment by ronycohen** [comment]
Hi, I found a proper error : 

When : 
```docker pull ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.4```

Failed to register layer: error creating overlay mount to /var/lib/docker/overlay2/a0556c1c16a7e94210560c8a324c8f65c1ed40bfa68e6b115e35b954b5551dca/merged: too many levels of symbolic links

**Comment by ronycohen** [comment]
I solved the overlay problem.

`sudo apt-get update && sudo apt-get upgrade`
Then 
`docker pull ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.4`
worked 

But still I have the 

```
23-02-14 11:53:25 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
23-02-14 11:53:58 ERROR (SyncWorker_2) [supervisor.docker.interface] Can't install ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.4: 404 Client Error for http+docker://localhost/v1.42/images/ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.4/json: Not Found ("No such image: ghcr.io/home-assistant/raspberrypi4-homeassistant:2023.2.4")
23-02-14 11:53:58 WARNING (MainThread) [supervisor.homeassistant.core] Error on Home Assistant installation. Retry in 30sec
```

**Comment by ikifar2012** [comment]
Strange, @pvizeli do you have any idea what could be causing this?

**Comment by raderarn** [comment]
I spent the whole day yesterday trying to install supervised. I have a MeLe quieter2q and downloaded the latest Debian installation, 11.7. There were a lot of errors in journalctl after installing the supervisor. Unfortunately, I did not save them, but one was 
"home assistant AttributeError: 'bool' object has no attribute 'signature'". But it looks similar to this issue.

Actually, I did this three times, with a little difference in the installation procedure, but with no luck.

So, in the end, I downloaded a full DVD installation media with Debian 11.0 instead. And chose not to use an apk mirror. After the installation, I changed sources.list to use the default mirror. I then installed the prerequisites and finally run the supervisor installer.

This time everything worked fine! Then I could update Debian to the latest. My suspicion is there is something with an update of the Python packages.
